.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -26,4 +19,11 @@ body:
     label: Additional Information
     description: Add any other context/information about the problem here.
   validations:
-    required: false
\ No newline at end of file
+    required: false
+- type: checkboxes
+  attributes:
+    label: Preflight Checklist
+    description: Please ensure you've completed the following.
```

Comment:
"following." would be better as "following:"

The rest looks fine.

Reply:
Thanks for catching!